client/reader/index.js

```diff
@@ -46,7 +46,7 @@ module.exports = function() {
 
 		// Feed full post
 		page( '/read/post/feed/:feed_id/:post_id', controller.legacyRedirects );
-		page( '/read/feeds/:feed/posts/:post', updateLastRoute, controller.feedPost );
+		page( '/read/feeds/:feed/posts/:post', updateLastRoute, controller.sidebar, controller.feedPost );
```

Comment:
We want to show the sidebar on 404, so we need to render (and hide) it even if it is not a 404.